src/main/java/com/cleanroommc/groovyscript/compat/mods/chisel/Carving.java

```diff
@@ -0,0 +1,185 @@
+package com.cleanroommc.groovyscript.compat.mods.chisel;
+
+import com.cleanroommc.groovyscript.api.GroovyLog;
+import com.cleanroommc.groovyscript.compat.mods.ModSupport;
+import com.cleanroommc.groovyscript.helper.ingredient.IngredientHelper;
+import com.cleanroommc.groovyscript.registry.VirtualizedRegistry;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.SoundEvent;
+import org.apache.commons.lang3.tuple.Pair;
+import team.chisel.api.carving.CarvingUtils;
+import team.chisel.api.carving.ICarvingGroup;
+import team.chisel.api.carving.ICarvingRegistry;
+
+import java.util.ArrayList;
+import java.util.Collection;
+
+public class Carving extends VirtualizedRegistry<Pair<String, ItemStack>> {
+
+    private static final ICarvingRegistry registry = CarvingUtils.getChiselRegistry();
```

Comment:
Can you reassign `registry` later on such as on reload or wherever in the loading process chisel would be ready?
`static final` assignment means if the first attempt returns null, the API becomes unusable.

Reply:
Just use `CarvingUtils.getChiselRegistry()` everywhere